tests/e2e/testnet-scripts/start-chain.sh

```diff
@@ -377,15 +377,16 @@ fi
 # poll for chain start
 if [[ "$USE_COMETMOCK" == "true" ]]; then
     set +e
-    until $BIN query block --type=height 0 --node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
+    until $BIN query block --type=height 1--node "tcp://$CHAIN_IP_PREFIX.$QUERY_IP_SUFFIX:26658"; do sleep 0.3 ; done
```

Comment:
This was printing a panic when you use 0, now you're waiting for the first block, right?